docs/notebooks/des-gaia.ipynb

```diff
@@ -0,0 +1,660 @@
+{
```

Comment:
<p>where did you get url from. From <a href="https://desdr-server.ncsa.illinois.edu/despublic/dr2_tiles/" target="_blank">https://desdr-server.ncsa.illinois.edu/despublic/dr2_tiles/</a>, right? Is there some smart way to get list of names of files?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>There is not a simple name I'm aware about. I could provide some code to download the data, do you think we need it here?</p>

Reply:
I think it is ok. Maybe in the future.